HabboHotel/Rooms/RoomData.cs

```diff
@@ -33,7 +33,11 @@ public class RoomData
         Category = category;
         Description = description;
         Tags = new List<string>();
-        foreach (var tag in tags.Split(',')) Tags.Add(tag);
+        foreach (var tag in tags.Split(','))

```

Comment:
I would just change this to a single linq query: `Tags = tag.Split(',').Where(s => !string.IsNullOrEmpty(s))`

Also prefer `string.` over `String.`

Reply:
Does this resolve #66 ?